canonical-websites/build.snapcraft.io
# Add setup script to help developers setting up BSI

## Done

Add a setup script to help you building the environment file.
DONT MERGE BEFORE:
- run script is added to the project
- This PR is merged: https://github.com/canonical-webteam/generator-canonical-webteam/pull/52 

## QA

- `./setup.sh`
- `./run`
- [0.0.0.0:8083](0.0.0.0:8083)
- Check if you can navigate BSI. 

## TODO

- [x] Make sure wich variable for extra ports
